### ISSUE TITLE
Unique page title – Landing behaviour 

### ISSUE THREAD

**Issue by samsadasivan** [cds-snc/platform-forms-client#5830]
**_Specs: Windows 
Screen Readers Used: JAWS, NVDA 
Browsers Tested: Google Chrome, Microsoft Edge, Internet Explorer_** 

**_Specs: MacOS
Screen Readers Used: Voiceover
Browsers Tested: Safari_**

**Observation:** 
This behaviour is observed on all the pages. 

This form consists of six pages, all sharing the same title: “Accessibility Audit – ThoughtStorm”. 

It was also observed that upon landing on a new page, Screen Readers (SR) do not announce the page title but instead announce the first interactive element. Additionally, heading levels are skipped. 

Please refer to the video clip attached below for your review. 

**_Severity: Medium_** 

**Recommendation:**  
Assign a unique and descriptive title to each page that accurately reflects its content. 

Example:  

1. Form name – Applicant Information 

2. Form name – Eligibility Criteria 

3. Form name – Application Details 

4. Form name – Additional Information - Funding Support 

5. Form name – Repeating Sets 

6. Form name – Review your answers before submitting the form 

Upon landing on a new page, Screen Readers (SR) should announce the page title, skip to the main content - link, Logo and header and then read the heading followed by the page content. 

------
**_Notes from macOS test_** 

**Observation:** 
This behaviour is observed on all the pages. 

This form consists of six pages, all sharing the same title: “Accessibility Audit – ThoughtStorm”. 

It was also observed that upon landing on a new page, VoiceOver do not announce the page title but instead announce the first interactive element. Additionally, heading levels are skipped. 

Please refer to the video clip attached below for your review.

**Comment by thiessenp-cds** [comment]
Technically the forms-form app does have a descriptive page title. Arguing that it does not is quite subjective. For multi-page forms I agree that when a new "page" is shown after clicking "continue" the intention of the update is a new "page". So updating the meta title definitely makes sense. The above example of how to structure the page title "{Main site title} - {Current page title}" is excellent.
